modules/dbparser/dbparser-grammar.ym

```diff
@@ -101,6 +103,19 @@ parser_db_opts
 parser_db_opt
         : KW_FILE '(' string ')'                		{ log_db_parser_set_db_file(((LogDBParser *) last_parser), $3); free($3); }
 	| KW_DROP_UNMATCHED '(' yesno ')'			{ log_db_parser_set_drop_unmatched(((LogDBParser *) last_parser), $3); };
+        | KW_PROGRAM_TEMPLATE '(' template_content ')'
+          {
+             log_db_parser_set_program_template_ref(last_parser, $3);
+          }
+        | KW_MESSAGE_TEMPLATE '(' string ')'    {
+                                                  LogTemplate *template;
```

Comment:
why doesn't this use template_content?

Reply:
It is a nice simplification indeed. Thanks.